cmd/virt-handler/virt-handler.go

```diff
@@ -342,6 +346,10 @@ func (app *virtHandlerApp) Run() {
 
 	go vmController.Run(10, stop)
 
+	nodeLabellerController := nodelabeller.NewNodeLabeller(&devicemanager.DeviceController{}, app.clusterConfig, nodeSharedInformer, app.virtCli, app.HostOverride, app.namespace)
+
+	go nodeLabellerController.Run(10, stop)
```

Comment:
practically it doesn't matter much, but node labeller only works on a single key. it only needs a single thread